Models/library.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require_relative 'author'
+require_relative 'book'
+require_relative 'order'
+require_relative 'reader'
+require 'yaml'
+
+class Library
+  attr_reader :books, :orders, :authors, :readers
+
+  def initialize(authors, books, readers, orders)
+    @authors = authors
+    @books = books
+    @readers = readers
+    @orders = orders
+  end
+
+  def read_from_yaml(file_name = 'library.yml')
+    if File.exist?(file_name)
+      library = YAML.load_file(file_name)
+      @authors = library.authors
+      @books = library.books
+      @orders = library.orders
+      @readers = library.readers
+    else
+      puts 'There is no such file.'
```

Comment:
PUTS should not be in the working files of the app.
They can only be used in a class-interface that communicates with the user.
 In our case, this is the main file
If File doesn`t exist can raise Error

Reply:
Thanks. puts was deleted